
Common Test starting (cwd is /home/runner/work/eturnal/eturnal)



CWD set to: "/home/runner/work/eturnal/eturnal/_build/test/logs/ct_run.nonode@nohost.2022-01-03_11.07.59"

TEST INFO: 1 test(s), 16 case(s) in 1 suite(s)

Testing lib.eturnal: Starting test, 16 test cases

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
eturnal_SUITE:stun_tcp_auto failed on line 222
Reason: {badmatch,false}
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Testing lib.eturnal: *** FAILED test case 14 of 16 ***

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
eturnal_SUITE:stun_tls_auto failed on line 231
Reason: {badmatch,false}
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Testing lib.eturnal: *** FAILED test case 15 of 16 ***
Testing lib.eturnal: TEST COMPLETE, 14 ok, 2 failed of 16 test cases

Updating /home/runner/work/eturnal/eturnal/_build/test/logs/index.html ... done
Updating /home/runner/work/eturnal/eturnal/_build/test/logs/all_runs.html ... done
